src/devices/Board/I2cDeviceManager.cs

```diff
@@ -0,0 +1,131 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Device.I2c;
+using System.Text;
+
+namespace Iot.Device.Board
+{
+    internal class I2cDeviceManager : I2cDevice
```

Comment:
So this is maybe where we should as well think of #1335?

Reply:
We could add it here, but I think it would better be one level up, since this is just an extended version of the I2cDevice class.

Reply:
If we have the level up, then not sure we need this one. At the same time, that can do the work the time to have the higher level which can anyway be added.